### DISCUSSION TITLE
How to fix CORS errors?

### DISCUSSION THREAD

**Discussion by vincent157664** [vincent157664/auxilar_backend#3] [Q&A] [answered]
Hello everyone. How are you?

I am a full-stack developer who is working on ChatGPT extension. While developing, for testing I've deployed the project using Ngrok. After that I've faced CORS error. I can't get response as json. The response is HTML document. Please help me.

**Comment by kohhash** [comment]
I think you have error in extension code.
Is it chrome extension?

**Comment by vincent157664** [comment]
Yes, it is.

**Comment by kohhash** [comment]
If so, you need to share the code to integrate with backend.

**Comment by vincent157664** [comment]
I don't know which parts cause CORS error.

**Comment by kohhash** [comment, marked as answer]
```
If so, you should keep in this format in integrating with backend.
fetch(
        "https://<URL>",
        {
          method: "POST",
        }
      )
        .then((response) => response.json())
        .then((data) => {
          //process data here
        })
        .catch((error) => {
          console.log(error);
        });
```
Please try.
Thanks.

**Comment by vincent157664** [comment]
OK, thanks, I will test.

**Comment by kohhash** [comment]
I think you are really facing with not CORS error.

**Comment by vincent157664** [comment]
I've found error. Thanks.
